pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -849,7 +849,10 @@ func TestUpdateNodeStatusWithLease(t *testing.T) {
 	// Since this test retroactively overrides the stub container manager,
 	// we have to regenerate default status setters.
 	kubelet.setNodeStatusFuncs = kubelet.defaultNodeStatusFuncs()
-	kubelet.nodeStatusReportFrequency = time.Minute
+	// You will add up to 50% of nodeStatusReportFrequency of additional random latency for
+	// kubelet to determine if update node status is needed due to time passage. We need to
+	// take that into consideration to ensure this test pass all time.
+	kubelet.nodeStatusReportFrequency = 30 * time.Second
```

Comment:
This is the change to defalke `TestUpdateNodeStatusWithLease `.

Reply:
did you run this test like 100 times in a loop locally?

Reply:
this change makes sense to me. Once you run it many times in a row, please ping

Reply:
@mengqiy @SergeyKanzhelev for flake proof we ask people to use the `stress` tool, for reference https://pkg.go.dev/golang.org/x/tools/cmd/stress

```
go test -c -race ./pkg/kubelet/
stress ./kubelet.test -test.run=TestUpdateNodeStatusWithLease
5s: 145 runs so far, 0 failures
10s: 337 runs so far, 0 failures
15s: 536 runs so far, 0 failures
20s: 738 runs so far, 0 failures
25s: 938 runs so far, 0 failures
30s: 1160 runs so far, 0 failures
35s: 1365 runs so far, 0 failures
40s: 1562 runs so far, 0 failures
45s: 1766 runs so far, 0 failures
```

This /lgtm

Reply:
I run `go test -v -count=1 ./pkg/kubelet/...` 100 times locally in a loop. All are passing.

Reply:
is not the same, we should require proof with `stress` for flakes,